### PR TITLE
Added translations and removed unused entries

### PR DESCRIPTION
**Pull request by ig-ealbert**
I pulled the latest from Umbraco-CMS/dev-v7 and created a new branch off of it.  I then added the translations from Pull Request #1036, which was closed.  These changes still apply to dev-v7.  I hope it was correct, and easiest for you, to create a new pull request like this.

Changes include new translations, fixing existing translations, and removing unused entries to match the en.xml file.

### CODE REVIEW

File: src/Umbraco.Web.UI/umbraco/config/lang/ja.xml [hunk 1]
```diff
@@ -796,21 +803,13 @@ Runwayをインストールして作られた新しいウェブサイトがど
     <key alias="noColors">設定済みの色はありません。</key>
   </area>
   <area alias="relatedlinks">
-    <key alias="addExternal">外部リンクを追加</key>
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI/umbraco/config/lang/ja.xml]: These should not be removed, they're still used in v7

**Comment by nul800sebastiaan** [comment]
I couldn't use the other PR as you removed entries that were still used, can you review those please?

**Comment by ig-ealbert** [comment]
What is the base Umbraco language?  I was comparing with this file:
https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web.UI/umbraco/config/lang/en.xml
If you search for "relatedlinks" here, there are only 7 child keys.
If the en.xml file is wrong, I can add the entries in this section back into the ja.xml file.

**Comment by nul800sebastiaan** [comment]
@ig-ealbert Ha! You're absolutely right, I don't know where I got that information from, I must have been dreaming. ;-)

**Comment by nul800sebastiaan** [comment]
Thanks again, all merged in now!

**Comment by ig-ealbert** [comment]
Thank you for all your help with this process!

**Pull request merged**